prefoil/airfoil.py

```diff
@@ -150,7 +149,7 @@ def getTE(self):
 
         Returns
         -------
-        TE : 2darry [2]
```

Comment:
Is 2darray just not a real "type"?

Reply:
Is there a list of valid types somewhere that we use?

Reply:
I don't think there's a list